### PR TITLE
Activating flake8 again and tons of fixes to make it pass.

### PR DESCRIPTION
**Pull request by rockdog**
This change activates flake8 again with a max-line-limit of 120 and includes tons of fixes to make it actually pass. 

Maybe a max-line-limit of 80 (or 79) is preferable but before I‘m starting that I‘d like to get some feedback.

Anyway I think this change set is already worth merging.

Sven

### CODE REVIEW

**Comment by asottile** [comment]
We should add .pre-commit-config.yaml for this project :)

Add that and :shipit:

**Comment by rockdog** [comment]
Great. I activated flake8 in the tox.ini because I forgot to commit that change with my last commit. 

I also added pre-commit as suggested. In .pre-commit-config.yaml I did not add the max-line-limit to 120. So if someone modifies a file it will take the default of 79.

File: tox.ini [hunk 1]
```diff
@@ -7,32 +7,32 @@ setenv =
...
-#	flake8 pushmanager
-#	flake8 pushmanager/tests
-#	flake8 setup.py
+    testify {posargs:pushmanager.tests}
+    pyflakes pushmanager
+    pyflakes pushmanager/tests
+    pyflakes setup.py
+    flake8 --max-line-length=120 pushmanager
+    flake8 --max-line-length=120 pushmanager/tests
+    flake8 --max-line-length=120 setup.py
```
Comment (asottile) [inline tox.ini]: Add a section:

```
[flake8]
max_line_length 120
```

And then both flake8 and pre-commit will use it :)

**Comment by asottile** [comment]
fix, rebase (you conflict) + :shipit:

File: tox.ini (hunk 1 above)
Comment (asottile) [inline tox.ini]: Oh also, flake8 calls pyflakes so there's no reason to run both of them :)

**Comment by rockdog** [comment]
I resolved all conflicts, added the flake8 section to tox.ini and Travis CI is happy. I just can‘t push it since I only have read access to our repos :(

**Pull request merged**